code/api/src/migrations/6-order.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('orders', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      userId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'users',
+          key: 'id'
+        },
+        allowNull: false
+      },
+      crateId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'crates',
+          key: 'id'
```

Comment:
Thank you for explaining the difference between the id and crateId and the difference between the primaryKey and other keys.